share/lib/hoc/import3d/import3d_gui.hoc

```diff
@@ -1003,8 +1004,13 @@ func set_nameindex() {local i, min  localobj sec
 	return min
 }
 
+/*!
+ * @param $o1 Cell object (can be nil)
+ * @param $o2 (optional) List object where to store the hoc execute commands 
```

Comment:
* @param $o2 (optional) On return it will reference a list of strings  that were executed by this method.

Reply:
If the arg does not appear, be sure to destroy the 'commands'  contents.

Reply:
Going with the second option. Thank you